### PR TITLE
[4.0] Form fields prepared statements, Part 1

### PR DESCRIPTION
**Pull request by SharkyKZ**
Pull Request for Issue # .

### Summary of Changes

Adds prepared statements and cleans up queries in some form fields.

### Testing Instructions

Install testing sample data. Go to backend.

1) Go to Tags list. Inspect `Select Tag Type` field.
2) Go to articles list. Inspect `Select Author` field.
3) Edit a module. Inspect `Module Style` field.
4) Go to Articles configuration. Inspect `Choose a Layout` field.
5) Go to User manager, click on Permissions icon. Inspect `Select Component` field.
6) Create Tagged Items menu item. In the form inspect `Select Content Type` field.
7) Go to Template Manager. Edit a template file. Check that CodeMirror editor appears.

### Expected result

Works like before.

### Documentation Changes Required

No.

### CODE REVIEW

File: libraries/src/Form/Field/ChromestyleField.php [hunk 1]
```diff
@@ -234,11 +235,21 @@ protected function getTemplates()
 		$query = $db->getQuery(true);
 
 		// Build the query.
-		$query->select('element, name')
-			->from('#__extensions')
-			->where('client_id = ' . $this->clientId)
-			->where('type = ' . $db->quote('template'))
-			->where('enabled = 1');
+		$query->select(
```
Comment (Quy) [inline libraries/src/Form/Field/ChromestyleField.php:238]: Change to `$query->select($db->quoteName(['element', 'name']))`

Comment (SharkyKZ) [reply libraries/src/Form/Field/ChromestyleField.php:238]: Apparently, currently this is slower than having multiple `quoteName()` calls with strings as arguments.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 02554be8ef794712d80b04a5159354686cacf4e6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26063">issues.joomla.org/tracker/joomla-cms/26063</a>.</sub>

**Comment by tushar33** [comment]
I have tested this item :red_circle: unsuccessfully on e5e5f8a972f6305c305d3902d37695e9bf5f620b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26063">issues.joomla.org/tracker/joomla-cms/26063</a>.</sub>

**Comment by SharkyKZ** [comment]
@tushar33 What issue are you getting?

**Comment by tushar33** [comment]
Didn't get tags in article<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26063">issues.joomla.org/tracker/joomla-cms/26063</a>.</sub>

**Comment by SharkyKZ** [comment]
@tushar33 that's expected as explained in your issue https://github.com/joomla/joomla-cms/issues/26679.

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on f3f85742e7bd8ea466e5a391df4e310fd47eb796<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26063">issues.joomla.org/tracker/joomla-cms/26063</a>.</sub>

**Comment by jwaisner** [comment]
@Quy can you retest this PR?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26063">issues.joomla.org/tracker/joomla-cms/26063</a>.</sub>

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 47bb0c5908065aafeda52b1dff6f8226227da05b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26063">issues.joomla.org/tracker/joomla-cms/26063</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26063">issues.joomla.org/tracker/joomla-cms/26063</a>.</sub>

**Comment by HLeithner** [comment]
Thanks

**Comment by HLeithner** [comment]
Thanks

**Pull request merged**